core/new-gui/src/app/workspace/component/result-panel/visualization-frame/visualization-frame.component.ts

```diff
@@ -12,27 +15,47 @@ import { VisualizationFrameContentComponent } from "../../visualization-panel-co
   selector: "texera-visualization-frame",
   templateUrl: "./visualization-frame.component.html",
   styleUrls: ["./visualization-frame.component.scss"],
+  providers: [{ provide: NZ_ICONS, useValue: [FullscreenExitOutline, FullscreenOutline] }],
 })
 export class VisualizationFrameComponent {
   @Input() operatorId?: string;
+  @ViewChild("modalTitle") modalTitle!: TemplateRef<{}>;
```

Comment:
It may not be a good idea to use `!` to force the compiler not to perform a null check. read more [here](https://www.stevefenton.co.uk/blog/2018/01/typescript-definite-assignment-assertions/). I would suggest avoiding using it.